File: api/src/main/java/com/ruby/rubia_server/core/dto/CompanyDTO.java
package com.ruby.rubia_server.core.dto;

import com.ruby.rubia_server.core.enums.CompanyPlanType;
import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;

import java.time.LocalDateTime;
import java.util.UUID;

@Data
@Builder
@NoArgsConstructor
@AllArgsConstructor
public class CompanyDTO {
    
    private UUID id;
    private String name;
    private String slug;
    private String description;
    private String contactEmail;
    private String contactPhone;
    private String logoUrl;
    private Boolean isActive;
    private CompanyPlanType planType;
    private Integer maxUsers;
    private Integer maxWhatsappNumbers;
    private UUID companyGroupId;
    private LocalDateTime createdAt;
    private LocalDateTime updatedAt;
}

File: api-dotnet/Rubia.Server/DTOs/CompanyDto.cs
using Rubia.Server.Enums;

namespace Rubia.Server.DTOs;

public class CompanyDto
{
    public Guid Id { get; set; }
    public string Name { get; set; } = string.Empty;
    public string Slug { get; set; } = string.Empty;
    public string? Description { get; set; }
    public string? ContactEmail { get; set; }
    public string? ContactPhone { get; set; }
    public string? LogoUrl { get; set; }
    public bool IsActive { get; set; }
    public CompanyPlanType PlanType { get; set; }
    public int MaxUsers { get; set; }
    public int MaxWhatsappNumbers { get; set; }
    public int MaxAiAgents { get; set; }
    public Guid CompanyGroupId { get; set; }
    public DateTime CreatedAt { get; set; }
    public DateTime UpdatedAt { get; set; }
}